src/Chessboard/index.js

```diff
@@ -21,8 +21,7 @@ const ChessboardContext = React.createContext();
 
 const getPositionObject = position => {
   if (position === 'start')
-    // return fenToObj('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
-  return fenToObj('????R?r?/?????kq?/????Q???/????????/????????/????????/????????/????????');
+    return fenToObj('rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR');
```

Comment:
our initial position is all question marks

Reply:
yes, but as we haven't delete question mark functionality yet, we can make this point with another step after creating valid fen functionality.

Reply:
In that case, let's open an issue
https://github.com/chessfinder/chessboardjsx/issues/3